packages/orama/tests/insert.test.ts

```diff
@@ -4,6 +4,7 @@ import { Index } from '../src/components/index.js'
 import { getInternalDocumentId } from '../src/components/internal-document-id-store.js'
 import { AnyDocument, create, insert, insertMultiple, search } from '../src/index.js'
 import dataset from './datasets/events.json' assert { type: 'json' }
+import { safeArrayPush } from "../src/utils"
```

Comment:
This is not needed. in the test, we can use `push` because the number of the objects here is not so high

Reply:
Also: the build fails for this reason.

Reply:
done